jenkins/pyinstaller/v2/requirements.txt

```diff
@@ -6,3 +6,4 @@ dataclasses==0.7
 python-consul==1.1.0
 pathlib==1.0.1
 pytest==5.4.2
+ipaddress==1.0.23
```

Comment:
There is a process to be followed for getting a new third party component added, we need to follow that. I will share the same separately.

Reply:
Added new way 
sum(bin(int(x)).count('1') for x in netmask.split('.'))
to avoid import

Reply:
ok